### PR TITLE
Run Example DAGs on Cloud envs

### PR DESCRIPTION
**Pull request by pankajastro**
issue: #124
Add bash script to build the docker image and start the deployment
Add docker file for docker image
Master dag to trigger all example dag, collect a summary of run and send a report on slack

### CODE REVIEW

File: .circleci/integration-tests/master_dag.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                    if ti.state == "success":
+                        task_code = ":large_green_circle: "
+                    elif ti.state == "failed":
+                        task_code = ":red_circle: "
+                    elif ti.state == "upstream_failed":
+                        task_code = ":large_orange_circle: "
+                    task_message_str = f"{task_code} {ti.task_id} : {ti.state} \n"
+                    message_list.append(task_message_str)
+
+        print("".join(message_list))
```
Comment (phanikumv) [inline .circleci/integration-tests/master_dag.py:40]: Add log instead of print

File: .gitignore [hunk 2]
```diff
@@ -21,6 +21,14 @@ dev/logs/
 dev/connections.yaml
 dev/dags/
 
+# Integtation Tests
```
Comment (phanikumv) [inline .gitignore:24]: correct the typo here

Comment (pankajastro) [reply .gitignore]: Fixed 48f2b56790cc598041ed92114f377176d0e1f556

**Comment by pankajastro** [comment]
Currently, Databricks and CNCF examples are failing. Tracking that here databricks: https://github.com/astronomer/astronomer-providers/issues/159  CNCF: https://github.com/astronomer/astronomer-providers/issues/160

**Review by phanikumv — APPROVED** [review]
Good job Pankaj !

**Pull request merged**